src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -68,4 +68,9 @@ def handle_exception(e):
                 formatted_exception.status_code,
             )
 
+        @app.before_request
+        def log_request_info():
+            app.logger.info("Headers: %s", request.headers)
```

Comment:
what's the default logging level? Maybe we should print all logs when pf start --debug

Reply:
default level is info